schemas/ephys/Ephys Session Schema.json

```diff
@@ -0,0 +1,264 @@
+			"type": "string",
+			"title": "Experimenter full name",
+			"description": "First and last name of the experimenter."
+		},
+	"session_start_time": {
+		"type": "string", 
+		"format": "date-time",
+		"title": "Session start time"
+		},
+	"session_end_time": {
+		"type": "string", 
+		"format": "date-time",
+		"title": "Session end time"
+		},	
+	"specimen_id": {
+		"type": "integer",
+		"title": "Specimen ID"
+		},
+	"project_name": {
+		"type": "string",
+		"title": "Project name"
+	},
+	"project_id": {
+		"type": "string",
+		"title": "Project ID"
+		},
+	"session_type": {
+		"type": "string", 
+		"enum": ["Test", "Foraging A", "Spontaneous", "Foraging B"],
+		"title": "Session type"
+		},
+	"stimulus_protocol_id": {
+		"type": "string",
+		"title": "Stimulus protocol ID"
+		},
+	"rig_id": {
+		"type": "string",
+		"title": "Rig ID",
+		},
+	"probes": {
+		"type": "array",
+		"items": { "$ref": "#/$defs/probe" },
+		"uniqueItems": true,
+		"title": "Probes"
+		},
+	"lasers": {
+		"type": "array",
+		"items": {"$ref": "#/$defs/laser"},
+		"uniqueItems": true,
+		"title": "Lasers"
+		}
+	},
+	"$defs": {
+		"probe": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"type": "string",
+					"title": "Name"
+					},
+				"tip_targeted_structure": {
+					"type": "string"
+					},
+                "other_targeted_structures": {
+                    "type": "string"
+                },
+                "targeted_ccf_coordinates": {
+                    "type": "array",
+                    "items": {"$ref": "#/$defs/ccf_coords"},
+                    "uniqueItems": true,
+                    "minItems": 3,
+                    "maxItems": 3,
+                    "title": "Targeted CCF coordinates"
+                },
+                "targeted_lab_coordinates": {
+                    "type": "array",
+                    "items": {"$ref": "#/$defs/lab_coords"},
+                    "uniqueItems": true,
+                    "minItems": 3,
+                    "maxItems": 3,
+                    "title": "Targeted lab coordinates",
+                    "description": "Targeted coordinates relative to the headframe"
+                },
+                "manipulator_coordinates": {
+                    "type": "array",
+                    "items": {"$ref": "#/$defs/manipulator_coords"},
+                    "uniqueItems": true,
+                    "minItems": 3,
+                    "maxItems": 3,
+                    "title": "Manipulator coordinates"
+                },
+                "manipulator_angles": {
+                    "type": "array",
+                    "items": {"$ref": "#/$defs/manipulator_angles"},
+                    "uniqueItems": true,
+                    "title": "Manipulator angles"
+                }
+            },
+            "$defs": {
+                "ccf_coords": {
+                    "type": "object",
+                    "properties": {
+                        "direction": {
```

Comment:
Fine to go with this for now, but we should have Lydia look at `direction`.  CCF is opinionated about coordinate order and direction.

Reply:
LYDIA!!!

Reply:
is it version? reference key? what options should be available in the enum?

Reply:
I expect we will want a string enum that defaults to "CCFv3". But we should ask.